components/button/index.css

```diff
@@ -226,63 +216,56 @@
-			--spectrum-button-background-color-hover: var(--spectrum-transparent-white-400);
-			--spectrum-button-background-color-down: var(--spectrum-transparent-white-500);
-			--spectrum-button-background-color-focus: var(--spectrum-transparent-white-400);
+			--spectrum-button-background-color-default: var(--spectrum-transparent-white-100);
+			--spectrum-button-background-color-hover: var(--spectrum-transparent-white-200);
+			--spectrum-button-background-color-down: var(--spectrum-transparent-white-200);
+			--spectrum-button-background-color-focus: var(--spectrum-transparent-white-200);
 
-			--spectrum-button-content-color-default: var(--spectrum-white);
-			--spectrum-button-content-color-hover: var(--spectrum-white);
-			--spectrum-button-content-color-down: var(--spectrum-white);
-			--spectrum-button-content-color-focus: var(--spectrum-white);
+			--spectrum-button-content-color-default: var(--spectrum-transparent-white-800);
+			--spectrum-button-content-color-hover: var(--spectrum-transparent-white-900);
+			--spectrum-button-content-color-down: var(--spectrum-transparent-white-900);
+			--spectrum-button-content-color-focus: var(--spectrum-transparent-white-900);
 
 			&.spectrum-Button--outline {
 				--spectrum-button-background-color-default: var(--spectrum-transparent-white-25);
-				--spectrum-button-background-color-hover: var(--spectrum-transparent-white-400);
-				--spectrum-button-background-color-down: var(--spectrum-transparent-white-500);
-				--spectrum-button-background-color-focus: var(--spectrum-transparent-white-400);
-
-				--spectrum-button-border-color-default: var(--spectrum-transparent-white-400);
-				--spectrum-button-border-color-hover: var(--spectrum-transparent-white-500);
-				--spectrum-button-border-color-down: var(--spectrum-transparent-white-600);
-				--spectrum-button-border-color-focus: var(--spectrum-transparent-white-500);
+				--spectrum-button-background-color-hover: var(--spectrum-transparent-white-100);
+				--spectrum-button-background-color-down: var(--spectrum-transparent-white-100);
+				--spectrum-button-background-color-focus: var(--spectrum-transparent-white-100);
+
+				--spectrum-button-border-color-default: var(--spectrum-transparent-white-300);
+				--spectrum-button-border-color-hover: var(--spectrum-transparent-white-400);
+				--spectrum-button-border-color-down: var(--spectrum-transparent-white-400);
+				--spectrum-button-border-color-focus: var(--spectrum-transparent-white-400);
 			}
 		}
-
-		&.is-selected {
-			--spectrum-button-background-color-default: var(--spectrum-neutral-subdued-background-color-default);
-			--spectrum-button-background-color-hover: var(--spectrum-neutral-subdued-background-color-hover);
-			--spectrum-button-background-color-down: var(--spectrum-neutral-subdued-background-color-down);
-			--spectrum-button-background-color-focus: var(--spectrum-neutral-subdued-background-color-key-focus);
-		}
```

Comment:
I don't believe there's a "selected" state for buttons. That doesn't seem to an option in Figma, isn't represented anywhere in the [guidelines page](https://spectrum.adobe.com/page/button/), there's no "selected" state in the Storybook controls. We do have this block on `main` (actually, several is-selected selector blocks). 

Think it's ok to remove? I don't `is-selected` getting added to a button anywhere, but am I missing it in a component somewhere?

Reply:
I double checked the web component and you're correct that there is no selected state for button. Safe to remove!